### ISSUE TITLE
Some images fail to load

### ISSUE THREAD

**Issue by aubundy** [zero-to-mastery/breads-client#87]
<img width="571" alt="Screen Shot 2021-09-22 at 8 23 34 PM" src="https://user-images.githubusercontent.com/37122448/134442311-ff6e036d-6c63-4052-8d34-933e6b804a70.png">

Some images within an article card fail to load. When this happens, the alt text and a placeholder icon gets substituted for the original image. Whenever an image fails to load, we should prevent the alt text and placeholder from displaying. I'm not sure if we can prevent this, though. It may have something to do with [weserv](https://images.weserv.nl/), an image caching service we use

**Comment by mcflav** [comment]
I would like to take a look a this and maybe provide a solution. Just let me know what I need to do. thanks!

**Comment by aubundy** [comment]
Thanks @mcflav! The code for this can be found in `src/features/globalReadings/ListItem.tsx`. I think the issue may have something to do with `serveImageThroughCDN()`

**Comment by aubundy** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Image alternate text](https://trello.com/c/0oc4cqTZ/61-image-alternate-text)

**Comment by mcflav** [comment]
I just submitted the pull request for the changes you requested. Please review at your earliest convenience. Thanks.

![image](https://user-images.githubusercontent.com/52865683/137430004-8756568e-6073-47bf-a75f-85ef6bfd890c.png)
